app/controllers/api/projects.php

```diff
@@ -130,18 +140,50 @@
             }
         }
 
-        $databaseOverride = System::getEnv('_APP_DATABASE_OVERRIDE', null);
-        $index = array_search($databaseOverride, $databases);
+        $databaseOverride = App::getEnv('_APP_DATABASE_OVERRIDE');
```

Comment:
I think we are using the `System::getEnv` function now 🤔  Ive seen it in use in other places too